tutorials/migrating-rails-app-from-heroku-to-google-appengine.md

```diff
@@ -215,29 +214,26 @@ your database configuration.
           encoding: unicode
           pool: 5
           timeout: 5000
-          host: "/cloudsql/${DB_CONNECTION_NAME}"
+          host: "/cloudsql/[CONNECTION-NAME]"
           database: mydb
           username: postgres
           password: <%= ENV['DATABASE_PASSWORD'] %>
 
-    Again, make sure you replace `${DB_CONNECTION_NAME}` with your actual
-    database connection name.
-
-    Notice we're reading the password from an environment variable. We'll set
-    that up below when we deal with secrets.
-
+    Note that `password` refer to an environment variable. This is configured later
+    in the tutorial.
+    
 ### Configure the entrypoint
 
-The _entrypoint_ is the command run to start your application. For example,
+The _entrypoint_ is the command run to start your application, such as
```

Comment:
Now the modifier is misplaced. Maybe:

The _entrypoint_, a command such as `bundle exec rails s`, is used to start your application.

Reply:
Done.